chevah_build

```diff
@@ -138,8 +138,9 @@ BUILD_FOLDER='build'
 
 # Import shared code.
 . ./functions.sh
-# Import build environmet variables.
-. ./build-python-package/BUILD_ENV_VARS
+
+# Import build environment variables.
+. ./${BUILD_FOLDER}/BUILD_ENV_VARS
```

Comment:
In `./brink.conf` this is always `build`, so theoretically-speaking this might go out of sync.

But why all this toil to use a sub-dir? Let's just save to a file in the current dir and be done with it, `DEFAULT_VALUES` worked for years… :-]

Reply:
I didn't wanted to "pollute" the root dir and as a best practice have everying inside the build dir.